views/emails/html/new-comment.php

```diff
@@ -89,10 +86,10 @@
                         <tr>
                             <td>
                                 <p style="font-size: 16px; line-height: 30px; margin: 0; color: #7E8690;">
-                                    <span><?php _e( 'Comment', 'wedevs-project-manager' ); ?></span>
+                                    <span><?php esc_html_e( 'Comment', 'wedevs-project-manager' ); ?></span>
                                 </p>
                                 <div style="padding: 5px 15px; background: #edeef45e; border: 1px solid #f2f2f2; border-radius: 5px; margin-bottom: 30px; line-height: 26px; margin-top: 10px;">
-                                    <?php echo $content; ?>
+                                    Test Comment: <?php echo wp_kses_data( $content ); ?>
```

Comment:
Escape the string here and make it translatable as well.
`<?php echo esc_html__( 'Test Comment: ', 'wedevs-project-manager' ) . wp_kses_data( $content ); ?>`